<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidgetGestionnaire">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>1081</width>
      <height>791</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabProduit">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <attribute name="title">
      <string>Produit</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>661</width>
        <height>381</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="labelRayon">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Rayon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidgetRayon"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButtondeleteRayon">
             <property name="text">
              <string>&amp;Supprimer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddRayon">
             <property name="text">
              <string>&amp;Ajouter</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="labelCategorie">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Categorie</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxRayon"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidgetCategorie"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButtonDeleteCategorie">
             <property name="text">
              <string>Supprimer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAddCategorie">
             <property name="text">
              <string>Ajouter</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabPersonnel">
     <attribute name="title">
      <string>Personnel</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>81</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Employ√©s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>First name :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Last name :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>250</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Postal Code :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>190</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Country :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>280</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Town :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>320</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adress :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>400</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Email @ :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>500</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Type :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPrenom">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>110</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNom">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>150</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPays">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>190</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEmail">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>390</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonGestionnaire">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>480</y>
        <width>108</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Manager</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonControlleur">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>510</y>
        <width>108</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Controller</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>70</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Login :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLogin">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>70</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetEmploye">
      <property name="geometry">
       <rect>
        <x>52</x>
        <y>62</y>
        <width>531</width>
        <height>561</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>Nom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prenom</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>login</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date embauche</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButtonAddEmploye">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>590</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonModifierEmploye">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>590</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonSuppEmploye">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>640</y>
        <width>85</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPostalCode">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>240</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTown">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>280</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAdress">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>320</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAdress2">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>350</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>350</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adress2 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>540</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>s√©curit√© social number :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSSNb">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>540</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonClearPers">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>650</y>
        <width>97</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabPaye">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Fiche de paye</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>641</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>95</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Payer</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelGestionnaire">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gestionnaire</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="action_Quitter"/>
   </widget>
   <addaction name="menu_Fichier"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidgetGestionnaire</tabstop>
  <tabstop>lineEditLogin</tabstop>
  <tabstop>lineEditPrenom</tabstop>
  <tabstop>lineEditNom</tabstop>
  <tabstop>comboBoxPays</tabstop>
  <tabstop>lineEditPostalCode</tabstop>
  <tabstop>lineEditTown</tabstop>
  <tabstop>lineEditAdress</tabstop>
  <tabstop>lineEditAdress2</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>radioButtonGestionnaire</tabstop>
  <tabstop>radioButtonControlleur</tabstop>
  <tabstop>lineEditSSNb</tabstop>
  <tabstop>pushButtonAddEmploye</tabstop>
  <tabstop>pushButtonModifierEmploye</tabstop>
  <tabstop>tableWidgetEmploye</tabstop>
  <tabstop>pushButtonSuppEmploye</tabstop>
  <tabstop>listWidgetRayon</tabstop>
  <tabstop>pushButtonAddRayon</tabstop>
  <tabstop>listWidgetCategorie</tabstop>
  <tabstop>pushButtonDeleteCategorie</tabstop>
  <tabstop>pushButtonAddCategorie</tabstop>
  <tabstop>pushButtondeleteRayon</tabstop>
  <tabstop>comboBoxRayon</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButton_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
